app/javascript/mastodon/features/ui/index.js

```diff
@@ -16,6 +13,14 @@ import { refreshTimeline } from '../../actions/timelines';
 import { refreshNotifications } from '../../actions/notifications';
 import UploadArea from './components/upload_area';
 
+const staticColumns = { };
+
+const lazyLoadStaticColumns = () => {
+  import(/* webpackChunkName: "features/home_timeline" */ '../home_timeline').then(Component => staticColumns.HomeTimeline = Component.default);
+  import(/* webpackChunkName: "features/compose" */ '../compose').then(Component => staticColumns.Compose = Component.default);
+  import(/* webpackChunkName: "features/notifications" */ '../notifications').then(Component => staticColumns.Notifications = Component.default);
+};
+
```

Comment:
These `webpackChunkName` comments get repeated enough that I wonder if it wouldn't make sense to just create an `async_routes.js` or something that contains all the `import()s` in one place.